### PR TITLE
Dynamic Landing Page for Thank You Videos

### PR DESCRIPTION
**Pull request by beccaklam**
Created new dynamic landing page for thank you videos we are sending to donors who gave during end of year fundraising campaign for 2016.

@vojtech Can you add the custom events using this? Not sure how it works...

### CODE REVIEW

File: src/pages/jan-thank-you.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    var pageData = pageDataSet[this.props.route.pageType];
+    return (
+      <div className="jan-thank-you">
+        <div className={className}>
+          <ThankYouHeader thankYouSentenceId={pageData.thankYouSentenceId}/>
+          <div className="video-container">
+            <video width="600" height="480" controls>
+            {
+              pageData.videoSrc.map(function(videoSrc) {
+                console.log("video: ", videoSrc);
```
Comment (TheoChevalier) [inline src/pages/jan-thank-you.js]: you probably want to remove that line ;)

Comment (beccaklam) [inline src/pages/jan-thank-you.js]: Hi @TheoChevalier, I'm not sure what you mean. I think we need those lines...

Comment (TheoChevalier) [inline src/pages/jan-thank-you.js]: My comment was just for this one `console.log("video: ", videoSrc);` it’s just console output, right?

Comment (beccaklam) [inline src/pages/jan-thank-you.js]: @TheoChevalier Oh! Haha, yes, you are right :)

**Comment by TheoChevalier** [comment]
@beccaklam I wonder if we should use something like `max-width: 70%;` + auto margins on the title, I’m afraid it can look at bit weird on wide screens, what do you think?
(I’ve generated a loooong translation for this purpose)
capture d ecran - 2017-01-11 a 19 55 07

**Comment by beccaklam** [comment]
@TheoChevalier that's a good flag! I'll look into it!

File: src/pages/jan-thank-you.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+/* jshint -W067 */
+
+import React from 'react';
+import MozillaFooter from '../components/mozilla/footer.js';
+import ThankYouHeader from '../components/jan-thank-you-header.js';
+import analytics from '../lib/analytics.js';
+
+var pageDataSet = {
+  cute: {
+    videoSrc: ["https://d24kjznqej0s8a.cloudfront.net/2016/eoy/Cute/Cute_Final.mp4","https://d24kjznqej0s8a.cloudfront.net/2016/eoy/Cute/Cute_Final.webm"],
```
Comment (cadecairos) [inline src/pages/jan-thank-you.js]: Can you have the array items here and below on separate lines?

**Pull request merged**